### ISSUE TITLE
Droplist audits needed in The Boayahada Tree zone

### ISSUE THREAD

**Issue by TeoTwawki** [DerpyProjectGroup/topaz#975]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: Base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
The Korrigans are dropping Fiend blood and Carbuncles Rubies (leech items), the blood balls are dropping  ***Bloody Fang*** which is an Abyssea pop item dropped by bats in Abyssea. this is just 2 examples, there are more. The whole zones mobs need rechecked as something went bad here on upstream and I am guessing the same ID was mistakenly reused or overwritten. 

Edit: ok may not be _as bad_ as I was first thinking. Still kinda screwy tho.

**Comment by CatsEyeXI** [comment]
looking through 'good first issues' noticed this was still open but had a PR from hooksta... is there still any work required on this?

**Comment by TeoTwawki** [comment]
yes. ppl kept giving up on it when the realized the tedium involved on checking so many mobs in so many zones due to overlapping ids (or tired of fighting git itself).

this was the last attempt, one of the few that actually made it to a PR instead of my discord DMs https://github.com/LandSandBoat/server/pull/1129

**Comment by Metal36** [comment]
Do we trust FFXIDB as a reference for drop rate?

**Comment by Xaver-DaRed** [comment]
Much more than xiclopedia, but that would also depend on how much data there is

If FFXIDB has an entry with only 3 tries, that information is worthless, for instance

**Comment by Metal36** [comment]
Ok, for this zone there seems to be a lot of data. What's the minimum threshold of entries to establish a drop rate? 100? (most in this zone have well over 1k)

**Comment by TeoTwawki** [comment]
For this particular error in our tables it's going to be less about the drop rates than about mobs mistakenly sharing drop IDs / being assigned wrong drop IDs

Best judgement using available info adjusted to nearest of these as "no TH" is going to be accepted:
```
-- Variables
SET @ALWAYS = 1000;  -- Always, 100%
SET @VCOMMON = 240;  -- Very common, 24%
SET @COMMON = 150;   -- Common, 15%
SET @UNCOMMON = 100; -- Uncommon, 10%
SET @RARE = 50;      -- Rare, 5%
SET @VRARE = 10;     -- Very rare, 1%
SET @SRARE = 5;      -- Super Rare, 0.5%
SET @URARE = 1;      -- Ultra rare, 0.1%
```

**Comment by Xaver-DaRed** [comment]
100 is low, but better than nothing.

just remember to stablish drop rate, we use this chart, so the "average" column, nor the TH3+ in FFXIDB aren't useful.

![imagen](https://user-images.githubusercontent.com/60053999/174395480-96055876-222d-4cab-a9b8-e2bc04d74d7c.png)

This chart is useful becouse it contains the drop rates for TH0 TH1 and TH2, so it allows for a better judgement

**Comment by TeoTwawki** [comment]
anything over 1000 is a mistake, mostly likely someone didn't know that you can't do multiple drops that way where a mob can drop more than 1

**Comment by Metal36** [comment]
So, if I use FFXIDB as a reference for drop rate, and reasonably account for people not correctly inputting their data, should I round to the nearest variable number, or go with something closer to the reference data at FFXIDB? (123 for example, would round down to 100 uncommon)

**Comment by WinterSolstice8** [comment]
ffxidb's data was/is auto collected by the windower plugin of the same name, unfortunately without the raw data it's hard to conclude anything that has multiple drop slots

**Comment by Metal36** [comment]
Ok, I'm going through this zone. It'll be a great opportunity for me to learn.

**Comment by EpicTaru** [comment]
> Ok, I'm going through this zone. It'll be a great opportunity for me to learn.

Git ir dun! I'm here to help if you need it.

**Comment by TeoTwawki** [comment]
#2488
another bites the dust :cry:

**Comment by EpicTaru** [comment]
This is still on the back burner. My old laptop bit the big one and I'm still configuring my new one. Once that's done I'll test fixes @Metal36 gave me before he had to do an extended afk and then co-author a PR to close this issue.

**Comment by EpicTaru** [comment]
Or wait, does the pr you linked resolve this, Teo?

**Comment by EpicTaru** [comment]
Nvm, that pr was closed, I'll continue as I was with the testing of the fixes that were given to me

**Comment by TeoTwawki** [comment]
tl;dr mobs need split up to use separate drop IDs, but we don't know how many exactly got mixed up this way. 


PRs that start to do it tend to die, the one I linked died.

**Comment by EpicTaru** [comment]
Yeah, give me a few more ~~days~~ hours and I should be up and running on the new laptop to do testing and all then I'll do the PR to resolve the issue.  @Metal36 did good work in his resolutions, he just didn't have time to PR then before he had to AFK and turned it over to me to complete the testing and do the PR in his absence.
